ios/FluentUI/Table View/TableViewCell.swift

```diff
@@ -1116,9 +1101,7 @@ open class TableViewCell: UITableViewCell, TokenizedControlInternal {
 
     /// Initializes TableViewCell with the cell style.
     @objc public override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
-        let tokens = TableViewCellTokens()
-        self.tokens = tokens
-
+        self.tokenSet = TableViewCellTokenSet(customViewSize: { .default })
```

Comment:
I don't think this is right. I think this loses any change to `customViewSize`? It should be `slef.tokenSet = TableViewCellTokenSet(cutomViewSize: { self.customViewSize })` where `self.customViewSize` defaults to `.default`, or something, right?

Reply:
Good catch. It's a little tricky because you can't reference `self` before calling `super.init()` but I have a solution in mind.

Reply:
Clever. Is that better than making the token set lazy?

Reply:
Oh that's even smarter! You can't make it lazy in SwiftUI, but that's totally allowed in UIKit.